build.gradle

```diff
@@ -28,6 +28,9 @@ ext {
   organizationUrl = "http://interlok.adaptris.net"
   slf4jVersion = '2.0.3'
   mockitoVersion = '4.9.0'
+  
+  interlokCommonsOfflineJavadocDir = project.rootDir.getCanonicalPath() + "/interlok-common/build/docs/javadoc"
```

Comment:
I don't think we need this line and maybe not the one below either

Reply:
same as the comment above, saw it in interlok-core and thought it was one of the required changes to the build file for junit 5 and java 17